pkg/auth/providers/azure/clients/ms_graph_client.go

```diff
@@ -153,19 +152,14 @@ func msGraphTokenHasPermissions(token string) bool {
 	if !ok {
 		return false
 	}
-	permissionPresent := make(map[string]bool)
-	for _, p := range msGraphRequiredPermissions {
-		permissionPresent[p] = false
-	}
+	permissionSet := make(map[string]struct{})
```

Comment:
Question for you here - is this some go-ism (the struct{}) that I'm not aware of which essentially creates a set of values?

Reply:
Yeah, that's a quick way to make a set, simpler than a map of things to bools, a common pattern.